lib/logs.coffee

```diff
@@ -59,23 +59,37 @@ utils = require('./utils')
 #
 # deviceLogs.on 'error', (error) ->
 # 	throw error
+#
+# deviceLogs.on 'clear', ->
+# 	console.clear()
 ###
 exports.subscribe = (pubnubKeys, device) ->
-	channel = utils.getChannel(device)
+	{ channel, clearChannel } = getChannels(device)
 	instance = pubnub.getInstance(pubnubKeys)
 	emitter = new EventEmitter()
 
+	emit = (event, data) ->
+		emitter.emit(event, data)
+
+	onMessage = (message) ->
+		if message.channel is clearChannel
+			return emit('clear')
+
+		if message.channel is channel
+			extractMessages(message.message).forEach (payload) ->
+				emit('line', payload)
+
+	pubnubListener = message: onMessage
+
+	instance.addListener(pubnubListener)
```

Comment:
Ah, found something: `PNNetworkIssuesCategory` event type can be used in the status callback